services/brig/src/Brig/API/User.hs

```diff
@@ -373,21 +367,22 @@ createUser new = do
 
   createUserTeam <- do
     activatedTeam <- lift $ do
-      case (tid, newTeam) of
-        (Just tid', Just nt) -> do
-          created <- liftSem $ GalleyProvider.createTeam uid (bnuTeam nt) tid'
+      case (tid, mNewTeamUser) of
+        (Just tid', Just newTeamUser) -> do
+          liftSem $ GalleyAPIAccess.createTeam uid (bnuTeam newTeamUser) tid'
           let activating = isJust (newUserEmailCode new)
+              BindingNewTeam newTeam = newTeamUser.bnuTeam
           pure $
             if activating
-              then Just created
```

Comment:
Is this changing the behavior?

Reply:
I think this is one of the places where the return value from `GalleyAPIAccess` is missing, so we have to construct it here.  Otherwise I don't see any changes in the flow.